### DISCUSSION TITLE
would a PR for the "novalidate" and possibly  pattern attributes be accepted?

### DISCUSSION THREAD

**Discussion by roni-estein** [filamentphp/filament#662] [Ideas]
They could be dealt with by extraAttrrubutes but the default representation of `novalidate` is simply the word, and not `novalidate="novalidate"` or "true" or "on"

This is useful when overriding items like required for custom date fields that like a masked date of birth field ([ mm/dd/yyyy ] a convenient north american standard), where the actual data represented on the form is not the data being submitted. Not a big issue for livewire but it can make some mobile browsers render oddly, or ask let the browser html required open a bubble. This is also useful for some specific number masks.

Allowing a "novalidate /  required / pattern=<regex>" lets the user get the best UX while not forcing odd HTML secondary effects.

I'd add a novalidate function that mimics required to just print out novalidate. Pattern is another main MDN function though it will come out represented properly. Adding the pattern attribute, makes its intention more clean and the api appear more complete leave extra attributes for things that are more genuinely extra.

Thx.

**Comment by danharrin** [comment]
Yup, I'd accept those. On a similar note, I added `inputMode()` support yesterday.
